dashboard/test/mailers/previews/pd_workshop_mailer_preview.rb

```diff
@@ -165,6 +165,20 @@ def facilitator_pre_workshop_csd
       target: :facilitator
   end
 
+  def facilitator_pre_workshop_csf_intro
+    mail :facilitator_pre_workshop,
+      Pd::Workshop::COURSE_CSF,
+      Pd::Workshop::SUBJECT_CSF_101,
+      target: :facilitator
+  end
+
+  def facilitator_pre_workshop_csf_deepdive
+    mail :facilitator_pre_workshop,
+      Pd::Workshop::COURSE_CSF,
+      Pd::Workshop::SUBJECT_CSF_201,
+      target: :facilitator
+  end
```

Comment:
you mentioned not having tested the emails, but I also see that you added previews. just checking whether you have viewed these at http://localhost-studio.code.org:3000/rails/mailers ?

Reply:
yeah I have, and they work fine there! I also double checked the old csd previews to make sure I didn't break them